benchmarks/ner_embeddings/project.yml

```diff
@@ -0,0 +1,272 @@
+title: "Comparing embedding layers in spaCy"
+description: |
+  This project compares `MultiHashEmbed` with its standard embedding counterpart
+  `MultiEmbed` for Named Entity Recognition. In order to access the datasets,
+  you first need to clone the [`spancat-datasets`](https://github.com/explosion/spancat-datasets)
+  repository:
+
+  ```sh
+  # Clone the spancat-datasets repo to access the datasets
+  git clone git@github.com:explosion/spancat-datasets.git
+  ```
+
+  and run the necessary conversion scripts. For example, let's perform the
+  conversion command for the [Anatomical Entity Mention (AnEM)](http://www.nactem.ac.uk/anatomy/)
+  corpus:
+
+  ```sh
+  # While inside the spancat-datasets repository
+  spacy project run anem
+  ```
+
+  This will generate the spaCy files that you can use for training. Once done,
+  you **should copy these files to this project**. For example, you can perform
+  a directory copy in Linux via:
+
+  ```sh
+  cp -r spancat-datasets/corpus/ner/*. ner_embeddings/corpus/. 
```

Comment:
There's an additional step here in which a user needs to `cd ..` back to the parent directory before this command should be run.

Reply:
What about just removing this and saying it in the intro that use the `span-labeling-datasets` to preprocess the data sets and copy the `.spacy` files that you'd like to work with into `corpus/`? The reason why we are not running all of it automatically is because you might not want to preprocess all the datasets.

Reply:
Yeah, that makes sense. I think if you want to give the user the flexibility to select the datasets, the best thing to do would be to provide that generic message (use `span-labeling-datasets` and the commands there), and then still provide some example commands to move the data in here (e.g. `cp -r <your_span-labeling-datasets_folder> <this_dir/corpus/>`).